src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.Sdk.BeforeCommon.targets

```diff
@@ -92,6 +92,24 @@ Copyright (c) .NET Foundation. All rights reserved.
     <Reference Include="System.Xml"/>
   </ItemGroup>
 
+  <!-- Set the default versions of the NETStandard.Library or Microsoft.NETCore.App packages to reference.
+       The implicit package references themselves are defined in Microsoft.NET.Sdk.props, so that they can be overridden
+       in the project file. -->
+  <PropertyGroup Condition="'$(TargetFrameworkIdentifier)' == '.NETStandard' And '$(NetStandardDefaultPackageVersion)' == ''">
+    <NetStandardDefaultPackageVersion>1.6.0</NetStandardDefaultPackageVersion>
```

Comment:
I tried this at your suggestion, and it turns out that referencing version 1.6.1 causes a bunch of extra files to be published.  I've filed #571 for this.